.github/workflows/terragrunt-plan-staging.yml

```diff
@@ -18,18 +18,22 @@ env:
   AWS_ACCESS_KEY_ID: ${{ secrets.STAGING_AWS_ACCESS_KEY_ID }}
   AWS_SECRET_ACCESS_KEY: ${{ secrets.STAGING_AWS_SECRET_ACCESS_KEY }}
   CONFTEST_VERSION: 0.46.0
-  TERRAFORM_VERSION: 1.4.2
-  TERRAGRUNT_VERSION: 0.46.3
+  TERRAFORM_VERSION: 1.6.4
+  TERRAGRUNT_VERSION: 0.53.2
   TF_INPUT: false
-  TF_VAR_ecs_secret_token_secret: ${{ secrets.STAGING_TOKEN_SECRET }}
-  TF_VAR_recaptcha_secret: ${{secrets.STAGING_RECAPTCHA_SITE_SECRET}}
+  ECS_SECRET_TOKEN: ${{ secrets.STAGING_TOKEN_SECRET }}
```

Comment:
Yes and no.  In the new secrets module I was using environment variables for local values so that I could set defaults.  After testing and researching to find that TF_VAR's take precedence over local values I forgot to change these back.